src/components/TutorialBanner.vue

```diff
@@ -0,0 +1,59 @@
+<script>
+import { mapGetters } from 'vuex'
+
+export default {
+  props: {
+    pageScroll: {
+      type: Boolean,
+      required: false,
+      default: () => false
+    }
+  },
+  data() {
+    return {
+      dismissed: false
+    }
+  },
+  computed: {
+    ...mapGetters('data', ['flows']),
+    showBanner() {
+      let show = this.flows?.length === 0
+      if (!this.dismissed && this.pageScroll) {
+        show = !this.pageScroll
+      } else if (this.dismissed) {
+        show = false
+      }
+      return show
+    }
+  }
+}
+</script>
+
+<template>
+  <v-banner :value="showBanner">
+    <v-avatar slot="icon" color="#3b8dff" size="40">
+      <v-icon icon="mdi-lock" color="white">
+        school
+      </v-icon>
+    </v-avatar>
+    Pst! It looks like you don't have any flows yet; check out the
+    <a href="tutorial">tutorials</a> for walkthroughs on writing and registering
```

Comment:
The tutorial link should probably be using `router-link` so that we're using in-app navigation and the documentation link should open in a new tab (using the `ExternalLink` component)